config/generateShellEnv.coffee

```diff
@@ -9,7 +9,7 @@ module.exports.create = (KONFIG, options = {}) ->
   traverse.paths(KONFIG).forEach (path) ->
     node = traverse.get KONFIG, path
     return  if typeof node is 'object'
-    add "KONFIG_#{path.join('_')}".toUpperCase(), node
+    add "KONFIG_#{path.join('_').replace(/\./g, "_")}".toUpperCase(), node
```

Comment:
I've thought I've left a bug that didn't cause any issues yet :)

Well, most of the environment variables are not directly referenced because they are read into an object called KONFIG in node modules and respective struct types in golang packages during start phase of programs. It provides a way to set configuration in runtime without any configure or build process.